src/com/google/JsComp.gwt.xml

```diff
@@ -0,0 +1,12 @@
+<module rename-to="jscomp">
+  <inherits name="com.google.gwt.user.User" />
+  <inherits name="com.google.common.base.Base"/>
+  <inherits name="com.google.common.collect.Collect"/>
+  <source path="debugging/sourcemap" />
+  <source path="javascript/jscomp" excludes=".svn,.git,**/ant/**,**/refactoring/**,**/webservice/**,**/testing/**" />
+  <source path="javascript/rhino" excludes=".svn,.git,**/testing/**" />
+  <super-source path="javascript/jscomp/gwt/super" />
```

Comment:
The internal BUILD has rules to generate .gwt.xml files from dependency info. If possible I would like to get rid of this file and obtain something similar